src/index.js

```diff
@@ -0,0 +1,163 @@
+let left_red_channel, right_red_channel,img;
+
+readyScript();
+
+function changeScreen(event) {
+  var op = event.target.className;
+  if (op.includes("btn-next")) {
+    $(event.target).parents(".screen").fadeOut("slow");
+    $(event.target).parents(".screen").next().fadeIn("slow");
+
+    if($(event.target).parents(".screen").attr('class').includes("container-3")){
+      document.getElementById('stepTwo').appendChild(img);
+      $(".container-4 .btn-next").attr("disabled", true);
+      $('img#sel').imgAreaSelect({
+        handles: true,
+        autoHide:true,
+        onSelectEnd: cb
+      });
+    }
+    else if($(event.target).parents(".screen").attr('class').includes("container-4")){
```

Comment:
I would appreciate seeing this block of code split into more function if possible because it looks quite large and I kind of see a pattern :)

Reply:
Done!